labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -76,6 +88,6 @@ object CollectionExercise04 {
    * To keep it simple it's ok to use String.split to extract all words of a sentence.
    */
   def calcLengthLongestWord(lines: String*): Int = {
-    error("fix me")
+    lines.flatMap(_.split("\\W+")).map(_.length).max
```

Comment:
If you just use map, you end up with a list of lists of words. So you'd actually be finding the most words used in any one line.